src/index.js

```diff
@@ -145,16 +149,8 @@ export default (apiUrl, httpClient = fetchJson) => {
 
         // If there are multiple urls then process them in parallel
         if (Array.isArray(url)) {
-            let RESTResponse = httpClient(url[0], options).then(response =>
-                convertHTTPResponseToREST({
-                    response,
-                    type,
-                    resource,
-                    params
-                })
-            );
-            return Promise.all(
-                url.filter((_, i) => i > 0).map(singleUrl =>
+            const responses = await Promise.all(
```

Comment:
not quite following you can explain on a call maybe.

Reply:
same as previous implementation, just using async/await instead of promise return

Reply:
i understand now